scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -3,8 +3,8 @@
 
 declare namespace WebdriverIO {
     function remote(
-        options: object,
-        modifier?: () => void
+        options?: WebDriver.Options,
```

Comment:
actually this should be `options?: WebdriverIO.Options,`

Reply:
The common object for `options` argument [looks like as](https://webdriver.io/docs/gettingstarted.html#create-test-file):
```js
    {
        logLevel: 'error',
        path: '/',
        capabilities: {
            browserName: 'firefox'
        }
    }
```
It is more like `WebDriver.Options`:
```js
    interface Options {
        protocol?: string;
        hostname?: string;
        port?: number;
        path?: string;
        queryParams?: {
            [name: string]: string;
        },
        capabilities?: DesiredCapabilities;
        logLevel?: WebDriverLogTypes;
        logOutput?: string | NodeJS.WritableStream
        connectionRetryTimeout?: number;
        connectionRetryCount?: number;
        user?: string;
        key?: string;
    }
```
instead of `WebdriverIO.Options`:
```js
    interface Options {
        runner?: string,
        specs?: string[],
        exclude?: string[],
        suites?: object,
        maxInstances?: number,
        maxInstancesPerCapability?: number,
        capabilities?: WebDriver.DesiredCapabilities | WebDriver.DesiredCapabilities[],
        outputDir?: string,
        baseUrl?: string,
        bail?: number,
        waitforTimeout?: number,
        waitforInterval?: number,
        framework?: string,
        mochaOpts?: object,
        jasmineNodeOpts?: object,
        reporters?: (string | object)[],
        services?: (string | object)[],
        execArgv?: string[]
    }
```
Do you agree? @christian-bromann

Reply:
yes, the other options are wdio testrunner options